react_app/src/components/AppDisplay.js

```diff
@@ -0,0 +1,23 @@
+import GameContainer from "components/Game/GameContainer";
+import SignUpGizmo from "components/Auth/SignUp";
+import LoginGizmo from "components/Auth/Login";
+import { connect } from "react-redux";
+
+import React from "react";
+
+const AppDisplay = ({ isAuthenticated, newUser }) => {
```

Comment:
I moved this out of your `App` file because it needs to connect to the store and you don't really want to be doing that in the same file you're providing it in.